### ISSUE TITLE
When using external video player, it would be nice if there were an option to always play a quality with audio

### ISSUE THREAD

**Issue by kkuhle** [TeamNewPipe/NewPipe#1474]
- [x ] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x ] I checked if the issue/feature exists in the latest version.

I would be surprised if this wasn't already requested, but I couldn't find it.  I use MX player with 720p as my video quality.  This works most of the time, but occasionally 720p is not available and then 480p defaults.  It would be nice if it would go to 360p so we could have audio.  If this could at least be a configuration option, that would be great!

Thanks for all the work on this incredible app!

**Comment by theScrabi** [comment]
Most, if not all videos come with 720p with combined audio streams.

We should have a more inteligent selection system, that would prefere streams with combined audio for cases like this :/

**Comment by kkuhle** [comment]
I agree.  If I'm not mistaken, 720p (not 60 fps) always has audio from what I've noticed.  But occasionally I play videos that do not have a 720p version and it seems the 480p without audio is defaulted.

It seems like since we know what copies won't have audio maybe we could have the option to have it default to 360p or another quality with audio, if available.

Sounds like you are already aware of this.  Maybe an enhancement for a future version!

**Comment by theScrabi** [comment]
Kind of sounds good :)

The code in question can be found [here](https://github.com/TeamNewPipe/NewPipe/blob/dev/app/src/main/java/org/schabi/newpipe/util/ListHelper.java#L155), who ever wants to take a look at it feel free :)

**Comment by SameenAhnaf** [comment]
@litetex https://github.com/TeamNewPipe/NewPipe/issues/754 should be closed in favor of this.
